.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Tag on Github and Release to RubyGems
+on: [workflow_dispatch]
+
+jobs:
+  release:
+    name: Release to RubyGems
+    runs-on: ubuntu-latest
+    # Ideally, we would require successful tests before
+    # allowing to publish. Because the tests need to be
+    # in a separate workflow file, we can't reference it
+    # here.
+    # needs: test
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Tag on Github
+        run: |
+          VERSION=$(echo "puts Gem::Specification::load('keycloak_oauth.gemspec').version" | ruby)
```

You can also do this:

```suggestion
          VERSION=$(ruby -e "puts Gem::Specification::load('keycloak_oauth.gemspec').version")
```

It's just for your information. Both versions are fine.

---

I like `ruby -e` so I committed your suggestion